Source/SwiftLintFramework/Extensions/Configuration+Remote.swift

```diff
@@ -141,11 +141,11 @@ internal extension Configuration.FileGraph.FilePath {
     ) throws -> String {
         if let cachedFilePath = cachedFilePath {
             if taskDone {
-                queuedPrint(
+                queuedPrintError(
```

Comment:
Is the changed output stream for these messages an API change that should be mentioned in the CHANGELOG explicitly? Some users might refer to the output in their build/lint jobs.

Reply:
Yes, although I'd put this in the bug fixes section because the (unwritten?) policy has been for violations to be printed to stdout and any other logging to be printed to stderr so that you could always `swiftlint ... > outfile.txt` or `swiftlint --reporter json > outfile.json` and not have invalid formatting because there was some error/info logs mixed in with the reports.

Reply:
a0100ef